### PR TITLE
Shuffling Parities Problem

### PR DESCRIPTION
**Pull request by shubham-sharma-2323**
This PR is to solve issue no. #350 
I have added the solution of the problem , mentioned in the issue.

### CODE REVIEW

**Comment by ij007** [comment]
1) This commit also has hoola hoop problem which is not the part of the issue.
2) There is no file extension in shuffling parties problem

**Comment by shubham-sharma-2323** [comment]
> 1. This commit also has hoola hoop problem which is not he part of the issue.
> 2. There is no file extension in shuffling parties problem

srry it was my next issue. I have deleted that file now.

**Comment by ij007** [comment]
Point 2 is still not solved.

File: DSA/Cpp/Shuffling-Parities-Problem.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
+Here's the solution: 
+
```
Comment (ij007) [inline DSA/Cpp/Shuffling-Parities-Problem.cpp]: Please fix this line.

**Pull request closed without merging**